WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -15,7 +15,8 @@ struct InPersonPaymentsLearnMore: View {
         }
             .padding(.vertical, Constants.verticalPadding)
             .onTapGesture {
-                ServiceLocator.analytics.track(.cardPresentOnboardingLearnMoreTapped)
+                ServiceLocator.analytics.track(event: WooAnalyticsEvent.InPersonPayments
+                                                .cardPresentOnboardingLearnMoreTapped(countryCode: SiteAddress().countryCode))
```

Comment:
I think it would be better to use the `CardPresentConfiguration` here too. I know they're the same under the hood, but it makes us consistent and makes the code easier to change in future.

Since it's not being injected anyway, I'd suggest a new property:

```
let cardPresentConfiguration = CardPresentConfigurationLoader().configuration
```

And then to change this to
```suggestion
                                                .cardPresentOnboardingLearnMoreTapped(countryCode: cardPresentConfiguration.countryCode))
```

Reply:
Great suggestion! Updated in https://github.com/woocommerce/woocommerce-ios/pull/6517/commits/eaacdb92f877f293db8ba4fda3d2ffe7b430fe9e